src/main/java/com/google/cloud/tools/ide/login/GoogleLoginState.java

```diff
@@ -144,17 +160,18 @@ public HttpRequestFactory createRequestFactory() {
    * @throws IOException if something goes wrong while fetching the token
    */
   public String fetchAccessToken() throws IOException {
```

Comment:
@briandealwis One more note: `fetchAccessAccount()` isn't what you think (if I am not mistaken), which we will never use. `getActiveCredential()` is the one that returns a `Credential` to a client. (Not even `getActiveAccount()`.)